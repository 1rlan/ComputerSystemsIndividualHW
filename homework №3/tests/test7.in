34879138746901384769
