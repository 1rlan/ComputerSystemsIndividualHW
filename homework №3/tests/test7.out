3267296.775769
